test/remote/gateways/remote_plexo_test.rb

```diff
@@ -31,6 +31,22 @@ def setup
       description: 'Test desc',
       reason: 'requested by client'
     }
+
+    @network_token_credit_card = ActiveMerchant::Billing::NetworkTokenizationCreditCard.new({
+        first_name: 'Santiago', last_name: 'Navatta',
+        brand: 'Visa',
+        payment_cryptogram: 'UnVBR0RlYm42S2UzYWJKeWJBdWQ=',
+        number: '5555555555554444',
+        source: :network_token,
+        month: '12',
+        year: 2020
+      })
+  end
+
+  def test_successful_network_tokenization_request
```

Comment:
💬 Comment

Maybe a more clear name could be something like "test_successful_purchas_with_network_token" because a "tokenization request" is what a Token Requestor do to the Network.